deploy/olm-catalog/konveyor-operator/v1.2.0/konveyor-operator.v1.2.0.clusterserviceversion.yaml

```diff
@@ -820,6 +829,11 @@ spec:
       kind: MigStorage
       displayName: MigStorage
       description: A migration storage location
+    - name: migtokens.migration.openshift.io
+      version: v1alpha1
+      kind: MigToken
+      displayName: MigToken
+      description: A token for a given cluster
```

Comment:
just nit-picking but would it be better to have "A token to identify a user for a given cluster"?